### ISSUE TITLE
Error when editing configuration when searching for a role/channel

### ISSUE THREAD

**Issue by TTtie** [tt-bot-dev/tt.bot#88]
**Describe the bug**

**To Reproduce**
1. tt.config
2. Switch to a category requiring a channel/role
3. Try to set the role to something that has multiple items

**Expected behavior**
Ofc it works

**A few questions:**
- [x] This happens on official tt.bot
- [ ] This happens on a selfhosted instance

**If this happens on a selfhosted version, fill this out: **

**Node.js**: 
**tt.bot**: 

**Screenshots**

**Additional context**
```js
TypeError: msg.t is not a function
    at Query.start (C:\Users\TTtie\Desktop\tttie-bot\queries\query.js:13:49)
    at Object.module.exports [as channel] (C:\Users\TTtie\Desktop\tttie-bot\queries\queryChannels.js:17:25)
    at ConfigMenu.getCb (C:\Users\TTtie\Desktop\tttie-bot\commands\config.js:115:49)
    at process._tickCallback (internal/process/next_tick.js:68:7)
D
```

**Comment by aepic999** [comment]
You said tomorrow, 6 days ago? Why ain't this closed then?

**Comment by aepic999** [comment]
You said tomorrow, 12 days ago? Why ain't this closed then?

**Comment by TTtie** [comment]
@aepic999 because I am lazier than expected and working on other things

**Comment by aepic999** [comment]
it's been over a month now what the fuck

**Comment by TTtie** [comment]
Actually it was fixed already but not commited yet

**Comment by TTtie** [comment]
it is finally published: 0e978f29ca8c8aa21e811083d7dd108078cc2f80
